src/main/java/xbot/common/subsystems/pose/BasePoseSubsystem.java

```diff
@@ -46,14 +46,14 @@ public abstract class BasePoseSubsystem extends BaseSubsystem implements DataFra
     protected BooleanProperty rioRotated;
     protected boolean firstUpdate = true;
     protected double lastSetHeadingTime;
-    public static double fieldXMidpointInMeters = 8.2705;
+    public static double fieldXMidpointInMeters = 8.7785;
```

Comment:
⭐ while we're here, consider turning this into a Distance value of Meters.of instead
```suggestion
    public static Distance fieldXMidpoint = Meters.of(8.7785);

Reply:
Maybe we should also add a comment about what years field this is for? Maybe have a list of past lengths in case someone was running 2024?

Reply:
Where did we get this new value from? The field is relatively evergreen, so this one surprises me a little.

Reply:
The Game Manual specified the field length as approximately 26ft. 5in. by 57ft. 6(7/8)in. I took the length and converted it to meters then halved it.

Reply:
Can you verify this using AdvantageScope? If you drive the robot to 8.7785, is it exactly in the middle of the field?

Reply:
Yes, I verified it